packages/jsii/lib/assembler.ts

```diff
@@ -913,3 +983,9 @@ function _toDependencies(assemblies: ReadonlyArray<spec.Assembly>): { [name: str
     }
     return result;
 }
+
+interface DeferredRecord {
+    fqn: string;
+    depFqns: string[];
```

Comment:
Some doc comments would be welcome here... Is "dep" for "depended" or "depending" ?